daemon/libnetwork/drivers/bridge/port_mapping_linux.go

```diff
@@ -218,11 +191,8 @@ func (n *bridgeNetwork) sortAndNormPBs(
 //   - same host ports or ranges are adjacent, then
 //   - ordered by container IP (then host IP, if set).
 func cmpPortBindingReqs(a, b portmapperapi.PortBindingReq) int {
```

Comment:
This seems like a broadly useful function that would be worth moving into `portmapperapi`.
```go
func (pb PortBindingReq) Compare(other PortBindingReq) int
```